src/lib/http2Session.ts

```diff
@@ -0,0 +1,114 @@
+import * as makeDebug from 'debug'
+import * as http2 from 'http2'
+
+const debug = makeDebug('http2session')
+const DEFAULT_MAX_STREAMS = (2 ** 31) - 1
+const MAX_TOTAL_REQUESTS = Infinity
+
+enum ConnectState {
+  DISCONNECTED,
+  CONNECTING,
+  CONNECTED
+}
+
+export default class Http2Session {
+  private _authority: string
+  private _http2Opts: object
+  private _requests: number
+  private _totalRequests: number
+  private _maxRequests: number
+
+  private _connected: ConnectState
+  private _connectPromise: Promise<http2.ClientHttp2Session>
+  private _client: http2.ClientHttp2Session
+
+  public id: string
+
+  constructor (authority: string, http2Opts: object, maxRequests?: number) {
+    this.id = String(Math.random()).substring(2)
```

Comment:
What is this for?

Reply:
leftover from when I was debugging; will remove